boards/inga_common/board.c

```diff
@@ -0,0 +1,51 @@
+/*
+ * board.c - Board initiazilation for the INGA sensor node
+ * Copyright (C) 2016 Robert Hartung <hartung@ibr.cs.tu-bs.de>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
```

Comment:
Missing doxygen tags (also in other files)

Reply:
True that.